IBAnimatable/Animatable.swift

```diff
@@ -44,9 +44,14 @@ public protocol Animatable: class {
    */
   var force: CGFloat { get set }
 
+  /**
+   Animation function as a timing curve. (default value should be none)
+   */
+  var timingFunction: TimingFunctionType { get set }
+
 }
 
-public extension Animatable where Self: UIView {
+public extension Animatable {
```

Comment:
I hope Swift can support the syntax like `public extension Animatable where Self: UIView or UIBarItem` later.